hubble/cmd/observe/flows.go

```diff
@@ -552,6 +552,12 @@ func newFlowsCmdHelper(usage cmdUsage, vp *viper.Viper, ofilter *flowFilter) *co
 	filterFlags.Var(filterVar(
 		"interface", ofilter,
 		"Show all flows observed at the given interface name (e.g. eth0)"))
+	filterFlags.Var(filterVar(
+		"encrypted", ofilter,
+		"Show only flows that were encrypted (WireGuard/IPsec)"))
```

Comment:
```suggestion
		"Show only encrypted flows (WireGuard/IPsec)"))
```

Using the past tense here might be confusing: after decryption, plain-text packets are delivered to pod/node. These decrypted packets also "were encrypted" before, but I don't want users to expect to see them too with such filter.